README.md

```diff
@@ -4,31 +4,31 @@ Reusable pipeline components for CNP Azure DevOps pipelines
 ## Terraform workflow templates
 
 The template files below contain steps to add Terraform Init/Plan/Apply/Destroy tasks to a pipeline.
-    
-    ├── scripts                                  
+
+    ├── scripts
     ├── steps
     │   └── terraform-precheck.yaml             # Precheck stage tasks
     │   └── terraform.yaml                      # Terraform plan and apply stage tasks
-    ├── tasks   
-    ├── vars   
+    ├── tasks
+    ├── vars
 
 ### Reusing templates:
 1. Create the [required folder structure](#required-terraform-folder-structure) in your repository
-2. Add the cnp-azure-devops-libraries repository resource as below  
-   
+2. Add the cnp-azure-devops-libraries repository resource as below
+
    ```yaml
-   
-   resources:  
```

Comment:
please make unrelated formatting changes in a separate pull request or disable your IDE trimming unrelated changes

Reply:
👍